### ISSUE TITLE
Missing Shadows

### ISSUE THREAD

**Issue by jimbrads** [RobertBeckebans/RBDOOM-3-BFG#230]
I know that the readme says that due to filtering some shadows disappear, but this seams like a bug to me, and I couldn't find anyone else with the same issue.
![rbdoom-3-bfg-20150619-225643-002](https://cloud.githubusercontent.com/assets/12974157/8265892/694aecbc-16dc-11e5-876d-20999e7a6bcc.jpg)
![rbdoom-3-bfg-20150619-225657-003](https://cloud.githubusercontent.com/assets/12974157/8265893/78f06624-16dc-11e5-8301-1ab46961f5a2.jpg)
![rbdoom-3-bfg-20150619-230036-002](https://cloud.githubusercontent.com/assets/12974157/8265894/7de91266-16dc-11e5-8991-1baf5a525660.jpg)
![rbdoom-3-bfg-20150619-230026-001](https://cloud.githubusercontent.com/assets/12974157/8265895/81559320-16dc-11e5-8266-3485d9546894.jpg)
![rbdoom-3-bfg-20150619-230244-004](https://cloud.githubusercontent.com/assets/12974157/8265897/87d8a8b8-16dc-11e5-8ea7-95fdfcd13167.jpg)
![rbdoom-3-bfg-20150619-230234-003](https://cloud.githubusercontent.com/assets/12974157/8265898/8c8d94cc-16dc-11e5-8b79-11e3f330998b.jpg)

I'm using "RBDOOM-3-BFG-win32-20140511-git-f950769.7z"
Operating System:	Windows 7 Professional, 64-bit (Service Pack 1)
DirectX version:	11.0 
GPU processor:		GeForce GTX 970
Driver version:		353.06
Direct3D API version:	11
Direct3D feature level:	11_1
CUDA Cores:		1664 
Core clock:		1114 MHz 
Memory data rate:	7010 MHz
Memory interface:	256-bit 
Memory bandwidth:	224.32 GB/s
Total available graphics memory:	7848 MB
Dedicated video memory:	4096 MB GDDR5
System video memory:	0 MB
Shared system memory:	3752 MB
Video BIOS version:	84.04.1F.00.B6
IRQ:			Not used
Bus:			PCI Express x16 Gen3
Device Id:		10DE 13C2 36701458
Part Number:		G401 0010

**Comment by jimbrads** [comment]
are these the "Carmack Shadows"?  Are the shadows just something that can't work?  I feel this must be a dumb question since no one is responding.?

**Comment by RobertBeckebans** [comment]
It depends on the light sources. There are a few lights in the first level that are too big or have their light center way off. That way it was possible to simulate sun like shadows with stencil shadows but it does not work that way with shadow maps. If you switch the shelter then you can see shadows again with different light sources that don't have such a bad setting. It is especially annoying that it mainly happens in the first level :)
![rbdoom-3-bfg-20150624-102502-001](https://cloud.githubusercontent.com/assets/1278614/8325768/42c5fc48-1a5c-11e5-9916-c1154780c1df.png)

**Comment by RobertBeckebans** [comment]
I also recommend to use newer binaries. I made a few general small fixes for shadow mapping and you can skip the cinematic sequences.

**Comment by jimbrads** [comment]
Thanks for the information.  Very good to know.  I thought I did it wrong.  I think that this info was unclear in the readme.  It might be a good addition to the 10) KNOWN ISSUES section

I guess its not possible to just let the resolution of the shadow map to get ugly because it gets so large?  Like Skyrim does when you put the shadow draw distance too far?

A different point, I have noticed that an openAll build gives the sound much better sound positioning than DirectX, or whatever windows uses by default.  Compare the two builds side by side at something like a TV or any stationary source of sound and see if you agree.  To me its worth a mod just that.  The soft shadows knock it over the fence.  Thank you!
